Taxi/Taxi/Presenter/AddTaxiParty/InfoContainer.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  InfoContainer.swift
+//  Taxi
+//
+//  Created by JongHo Park on 2022/06/14.
+//
+
+import SwiftUI
+
+// MARK: - 정보 선택 영역 가이드 뷰
+extension AddTaxiPartyView {
+    struct InfoContainer: ViewModifier {
+        let title: String
+        let selectedInfo: String
+        let toggle: Bool
+        let onClick: () -> Void
+
+        func body(content: Content) -> some View {
+            VStack(alignment: .leading, spacing: 0) {
+                section
+                    .zIndex(2)
+                content
+                    .zIndex(1)
+                    .frame(height: toggle ? nil : 0)
+                    .contentShape(Rectangle())
+                    .clipped()
+            }
+        }
+
+        private var section: some View {
+            Button {
+                onClick()
+            } label: {
+                HStack {
+                    Text(title)
+                        .font(.system(size: 18, weight: toggle ? .bold: .medium, design: .default))
```

Comment:
design은 디폴트 값이니까 지워도 되지 않을까요?

```suggestion
                        .font(.system(size: 18, weight: toggle ? .bold: .medium))
```

Reply:
7388c9db95739a385cc6ba1e1cecf6489cbb603d 에서 반영했습니다